index.html

```diff
@@ -484,6 +490,115 @@
               layout="[[layoutClass]]"
               route="{{subRoute}}"></generic-page>
           </template>
+          <!-- LURES    ============================================   -->
```

Comment:
Yes, there is technical debt with this site.  And the info arch that was created and launched with v1 has no room for other boards or lures.  This is correct.

Yes I have a plan for the future of the site.  Yes it will include a different info arch and will need sub-pages which the current site doesn't allow for.

I am following my manager's direction to get the identified wiki info into our site ASAP so that we can close it down.  This means we will have an interim period of 'island' pages until the new info arch can be discussed and you+ODX can estimate how long it will take to make it work.

I can show you the new info arch on Friday or early next week if you'd like.